### ISSUE TITLE
Is 0.0 acceptable for hte l1 and l2 parameters of ActivityRegularization layer?

### ISSUE THREAD

**Issue by EFanZh** [keras-team/keras#9339]
[The documentation](https://keras.io/layers/core/#activityregularization) says l1 and l2 parameter of ActivityRegularization layer should be “positive float”, but the default values for them are both 0.0. Shouldn’t the documentation should say “nonnegative float”?

**Comment by ozabluda** [comment]
sure should.

**Comment by ozabluda** [comment]
I checked, and ActivityRegularization does not complain about negative values, which I think is a good thing, but leaving it undocumented for now.

**Comment by briannemsick** [comment]
Of course it must be non-negative.

total loss = loss(w)+ r||w|| where loss is [mse, ce, etc...], r is the activity regularizer constant, w is the weights, and || || is some norm.

If r is negative you can minimize the loss by setting all of the weights to infinity.

Regularizers are a penalty, which is why r >= 0

**Comment by ozabluda** [comment]
>If r is negative you can minimize the total loss by setting all of the weights to infinity.

not if loss(w) -> +inf faster. mse, ce aren't the only losses possible.
